crates/bevy_winit/Cargo.toml

```diff
@@ -22,12 +22,12 @@ bevy_window = { path = "../bevy_window", version = "0.8.0" }
 bevy_utils = { path = "../bevy_utils", version = "0.8.0" }
 
 # other
-winit = { version = "0.26.0", default-features = false }
-approx = { version = "0.5.0", default-features = false }
-raw-window-handle = "0.4.2"
+winit = { version = "0.27", default-features = false }
+approx = { version = "0.5", default-features = false }
+raw-window-handle = "0.5"
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
-winit = { version = "0.26.0", default-features = false }
+winit = { version = "0.27", default-features = false }
```

Comment:
Why is winit duplicated here?
As wasm doesn't add any new features, the duplicate entry should just be removed.

Reply:
Yeah, not entirely sure... I'll remove it unless anyone has any concerns about it.